common.py

```diff
@@ -43,6 +43,7 @@
 
 AV_DEFINITION_FILE_PREFIXES = ["main", "daily", "bytecode"]
 AV_DEFINITION_FILE_SUFFIXES = ["cld", "cvd"]
+AV_SKIP_CLEAN_OBJECTS = os.getenv("AV_SKIP_CLEAN_OBJECTS", "False")
```

Comment:
```suggestion
AV_SKIP_CLEAN_OBJECTS = int(os.getenv("AV_SKIP_CLEAN_OBJECTS", "0"))
```
`"False"` is truthy. This should make it more difficult to accidentally set a truthy value. I'm definitely open to other approaches though.

Reply:
`"False"` would still return `0` in this case:
![image](https://github.com/trussworks/bucket-antivirus-function/assets/4298119/04a0deed-fdac-4fdb-ac8a-bb0085b34df2)

But I have no problems with changing the env.
Just keep in mind that it would only accept numeric values, and a `"False"` or `"True"` would raise instead.

Reply:
In python, `str_to_bool` isn't a built-in. Where are you getting that defined?